lib/slim_lint/cli.rb

```diff
@@ -21,7 +21,7 @@ def initialize(logger)
     def run(args)
       options = SlimLint::Options.new.parse(args)
       act_on_options(options)
-    rescue => ex
+    rescue StandardError => ex
```

Comment:
Anything better than StandardError here?

Reply:
It's meant to be a catch-all, so `StandardError` is fine (and preserves existing behavior).